orbit/radius.py

```diff
@@ -105,10 +105,12 @@ def __init__(self, env=None, username=None):
             self.username = username
             self.token = self.mk_hash(username)
             self.expiry = datetime.utcnow() + timedelta(minutes=min_per_ses)
-
-            if db.ses_getby_username(username):
-                db.ses_delby_username(username)
-            db.ses_ins((self.token, self.username, self.expiry_ts()))
```

Comment:
why not use self.expiry_ts()

Reply:
no reason, just forgot that it existed. Will fix